static/scss/answers/cards/product-prominentvideo.scss

```diff
@@ -0,0 +1,75 @@
+.HitchhikerProductProminentVideo
+{
+  display: flex;
+  flex-direction: column;
+  font-size: var(--yxt-font-size-md);
+  line-height: var(--yxt-line-height-md);
+  background-color: var(--yxt-color-brand-white);
+  width: 100%;
```

Comment:
is there any way we can use flex things instead of width and height 100%?

Reply:
Ya, I can experiment with that

Reply:
All of our cards use height and width 100% at the top level so that they fill up the Answers component div. I believe removing it here would require us to set `display: flex` on the answers component div, which seems out of scope. 

I investigated the other usages of 100% and those were actually not needed due to the use of flex as you suggested